servicedirectory/service_directory_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicedirectory
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestServiceDirectory(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	t.Run("createNamespace", func(t *testing.T) {
+		buf := new(bytes.Buffer)
+		if err := createNamespace(buf, tc.ProjectID); err != nil {
+			t.Errorf("CreateNamespace: %v", err)
+			t.Skip("Skipping rest of tests because CreateNamespace Failed")
```

Comment:
```suggestion
			t.Fatalf("CreateNamespace: %v", err)
```

`Errorf` continues execution, `Fatalf` stops it.